calc.sh

```diff
@@ -13,10 +13,10 @@ echo "Enter your choice"
 read ch
 
 case $ch in
-1)sum=`expr $n1 + $n2`
+1)sum=`expr $n1 - $n2`
    echo "Sum ="$sum;;
 
-2)sum=`expr $n1 + $n2`
+2)sum=`expr $n1 - $n2`
```

Comment:
Can you please add a comment like bug fixng

Reply:
Thanks for the review comments. I have corrected it. Please review and approve.